13-collabs.Rmd

```diff
@@ -5,6 +5,248 @@
+
+## Issues should tell us:
+- What the problem is - including examples or screenshots to demonstrate the problem.
+- What solution(s) could address the problem?
+- Who might be assigned to addressing this problem?
+- Potentially what timeline or urgency the problem has.
+
+## Pull Requests should tell us:
+- What issue is the work addressing?
+- How is it addressing the issue?
+- What pitfalls exist?
+- What should be looked at and reviewed carefully.
+
+## Engaging in Code Review - as an author
+
+The only way to know if your analysis is truly reproducible is to send it to someone else to reproduce! That sentiment is at the heart of code review.
+
+@Parker2017 describes code review:
+
+> Code review will not guarantee an accurate analysis, but it’s one of the most reliable ways of establishing one that is more accurate than before.
+
+Not only does code review help boost the accuracy and reproducibility of the analysis, it also helps everyone involved in the process learn something new!
+
+An effective code review atmosphere is something that individuals and their team have to _commit_ to (pun intended). Effective code review brings so many benefits not only to your project quality but also your communication skills through fostering a learning atmosphere!
+
+In this chapter we will discuss the two sides of code review. Code review ideally includes at least two people: the author of the pull request and the reviewer of the pull request. Depending on your job context, we realize that sometimes authors have to become their own reviewers if code review is not something that can be prioritized by your institution or team.
+
+## Author responsibilities in code review
+
+The code review process begins with the creation of a pull request (which we practiced in the previous chapter). Successful and efficient code review is born out of quality communication, which is a skill set on its own. You can set up your reviewers (and yourself) up for success by knowing what basic information can help get the code review conversation going.
+
+```{r, fig.alt="Ruby has filed a very large pull request with a very short PR description:  I updated the project and added new files. She then requested Avi to review this PR. This leaves Avi with very little information to go on and he is very confused. Avi comments: Ummm… I’m a bit confused. Can you explain the context of these changes? This is a lot for me to try to follow.  I’m also unsure what kind of feedback you are looking for. ", out.width = "100%", echo = FALSE}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g33bf0789107_300_2369")
+```
+
+Even if you end up being the only person who will review your own code, writing these things out is still very helpful and highly recommended. It can help you spot problems you might not have otherwise seen and generally help you document better for future you!
```

Comment:
```suggestion
Even if you end up reviewing your own code, writing the following information out is still very helpful and highly recommended. It can help you spot problems you might not have otherwise seen and generally help you document your code better for future you!
```

Reply:
simplifying for readability